modules/assim.sequential/R/sda.enkf.R

```diff
@@ -529,13 +513,13 @@ sda.enkf <- function(settings, obs.mean, obs.cov, IC = NULL, Q = NULL) {
       ###-------------------------------------------------------------------### 
       
       ### no process variance -- forecast is the same as the analysis ###
-      if (!processvar) {
+      if (processvar==FALSE) {
         mu.a <- mu.f
-        Pa   <- Pf + Q
+        Pa   <- Pf + Q #Where are we getting Q here? Shouldn't it be just Pf?
         ### yes process variance -- no data
       } else {
         mu.a <- mu.f
-        Pa   <- Pf + solve(q.bar)
+        Pa   <- Pf + solve(q.bar)#Where are we getting q.bar here?
```

Comment:
probably want to define q.bar outside the loop so this doesn't crash if there's no data at first time step